tests/core_validation/test_possible_fragment_spreads.py

```diff
@@ -0,0 +1,150 @@
+from graphql.core.language.location import SourceLocation
+from graphql.core.validation.rules import PossibleFragmentSpreads
+from utils import expect_passes_rule, expect_fails_rule
+
+def error(frag_name, parent_type, frag_type, line, column):
+  return {
+    'message': PossibleFragmentSpreads.type_incompatible_spread_message(frag_name, parent_type, frag_type),
+    'locations': [{ 'line': line, 'column': column}]
+  }
+
+
+def error_annon(frag_name, parent_type, frag_type, line, column):
+  return {
+    'message': PossibleFragmentSpreads.type_incompatible_anon_spread_message(frag_name, parent_type, frag_type),
+    'locations': [{ 'line': line, 'column': column}]
+  }
+
+def same_object():
```

Comment:
hm that's strange, I swear that they were running locally but apparently they weren't...